src/components/Buttons/CustomButton.tsx

```diff
@@ -9,12 +9,17 @@ import { CustomButtonProps } from '@/domain/props/CustomButtonProps';
  * @param variant solid | bordered | flat
  */
 
-export function CustomButton({ children, ...attributes }: CustomButtonProps) {
+export function CustomButton({ children, color, ...attributes }: CustomButtonProps) {
   const { variant } = attributes;
 
   return (
     <Button
       {...attributes}
+      style={{
+        backgroundColor: color,
+        minWidth: '120px',
+        textAlign: 'center',
+      }}
```

Comment:
This should be gone. First you should not be using style, until explicitly neccessary, but tailwind classes + clsx function . Nevertheless please do not change the Custom Button component without asking/communicating your special use case because this would change everything in the app